src/xpk/core/blueprint.py

```diff
@@ -82,7 +105,7 @@ def create_a3_mega_blueprint(
     primary_vpc_name: str = "network1",
     gpu_subnets_name: str = "gpunets",
 ) -> CtkBlueprint:
```

Comment:
Does this let us to deploy multiple clusters with only difference of cluster_name/deployment_name?

Reply:
names are prefixed with cluster name(unique per project), so it should be working